src/ldf-client-ui.js

```diff
@@ -114,6 +114,7 @@ require('leaflet/dist/images/marker-shadow.png');
       // Initialize map
       this.map = L.map($map.get(0)).setView([52.517987721, 6.116665362], 8);
       this.mapLayer = L.geoJSON().addTo(this.map);
```

Comment:
Will this one even be needed with these changed?

Reply:
Yes, since I first add the features to a layer and then I add that layer to the LayerGroup. This helps me to add it to a layer on the map before its added to a layer group.

Reply:
Yes, since I first add the features to a layer and then I add that layer to the LayerGroup. This helps me to add it to a layer on the map before its added to a layer group.